docs/best-practices/code-standard.md

```diff
@@ -0,0 +1,235 @@
+
+It's a Node.js based tool, so the tool and imported rules should be added to the ``package.json`` file. And the code standard rules are defined in our `.stylelintrc.json`. More info about rules [on their website](https://stylelint.io/user-guide/rules/).
+
+### Code Linting
+
+Here is a list of the plugins for the most used IDEs among our team:
+
+* [VS Code plugin](https://marketplace.visualstudio.com/items?itemName=shinnn.stylelint)
+* [JetBrains IDEs plugin](https://plugins.jetbrains.com/plugin/9276-intellij-stylelint-plugin) (PHPStorm, WebStorm, IntelliJ IDEA, etc )
+
+
+### Auto-fixing
+
+stylelint's client allow the paramenter ``--fix`` to auto-fix most of the rules violations. Unfortunaltely, it seems auto-fix is not supported yet by any the IDE's plugins. The workaround would be to use the `stylelint CLI` as an external tool.
+
+**Not all rules have auto-fix**. We might need to fix manually some errors pointed by stylelint. It's possible to contribute to the library and add auto-fix to those rules.
+
+**VS Code**
+
+* [Instructions here](https://code.visualstudio.com/docs/editor/tasks)
+
+**JetBrains IDEs**
+
+1. Install stylelint: `npm install stylelint`
+1. Go to  `Preferences > External Tools` (or press `⌘,` on Mac)
+1. Click on `Add` icon (or press `⌘N` on Mac)
+1. Fill the form with following info:    
+   – Name: `Stylelint Auto-fix`    
+   – Program: `node_modules/.bin/stylelint`    
+   – Parameters: `$FilePath$ --fix`    
+   – Working directory: `$FileDir$`
+
+All checkboxes in the form are optional, you can check whichever you want.
+
+Optionally you set a shortcut of you preferences for the newly created External tool. Go to **IDE Settings | Keymap** and locate the tool. Then use the Add Keyboard Shortcut context menu to assign a shortcut.
+
+### Pre-commit hook
+
+It is already implemented on this repository as an example. We only need to make sure **stylelint CLI** is installed  by running `npm install stylelint`.
+
+## JavaScript
+
+The single chosen tool here is [ESLint](https://eslint.org/). It's another Node.js based tool with pluggable rules.
+
+
+### Code Linting
+
+* [VS Code plugin](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)
+
+* **JetBrains IDEs**: It is already built in, but you must enable it at **Preferences | Languages & Frameworks | JavaScript | Code Quality Tools | ESLint** .
+
+### Auto-fixing
+
+
+ESLint's integration with the IDE's also supports auto-fix.
+
+**Not all problems are auto-fixable**
+
+
+**VS Code**
+
+Once the linting plugin is installed the command pallete wil have a `Fix all auto-fixable problems` command that applies ESLint auto-fix resolutions to all fixable problems.
+
+**JetBrains IDEs**
+
+Once ESLint is enabled you just need to hit `Alt + Enter` on the ESLint warning and select the option **Fix current file with ESLint** from the contextual menu.
```

Comment:
@Alex-DA depending of the platform, os, etc. the `Alt + Enter` shortcut is not always the same one. 

I think it's safer to say, use `Double Shift` action and search for **Fix current file with ESLint**. So it's general for all the JetBrains IDEs.

Reply:
Fixed in https://github.com/mobilejazz/harmony-reference/pull/82/commits/17ef30f91a245bf29b8a8b37ac9c8d4f421e00c0